packages/core/phase2/lib/getOperationSequence/generateOperations/resultClassHandlers/handleJudgementWithFinalResult.ts

```diff
@@ -58,10 +58,7 @@ export const handleJudgementWithFinalResult: ResultClassHandler = ({
     operations.push(operation)
   }
 
-  if (
-    result.PNCDisposalType === 2060 &&
-    checkRccSegmentApplicability(aho, ccrId) === RccSegmentApplicability.CaseRequiresRccButHasNoReportableOffences
-  ) {
+  if (result.PNCDisposalType === 2060 && checkRccSegmentApplicability(aho, ccrId) === true) {
```

Comment:
or `rccSegmentIsApplicable` - might be worth trying to understand what it's actually checking for and use that to determine what to call it since `applicable` is a bit opaque